ChangeLog

```diff
@@ -6,7 +6,7 @@ What's New in astroid 2.7.0?
 ============================
 Release Date: TBA
 
-
+* Added support to infer return type of typing.cast()
```

Comment:
Can you move this to the `2.6.3` section? You probably need to fetch the latest changes first.